app/views/play/CampaignView.coffee

```diff
@@ -431,7 +431,7 @@ module.exports = class CampaignView extends RootView
           @openModalView new CodePlayCreateAccountModal()
       else if me.get('name') and me.get('lastLevel') in ['forgetful-gemsmith', 'signs-and-portents', 'true-names'] and
       me.level() < 5 and not (me.get('ageRange') in ['18-24', '25-34', '35-44', '45-100']) and
-      not storage.load('sent-parent-email') and not me.isPremium()
+      not storage.load('sent-parent-email') and not me.isPremium() and not me.isStudent()
```

Comment:
Do we care about teachers here or any other user types?  Should we whitelist to only home users?

Reply:
This is a pretty good point. I'll add a `not me.isTeacher()` and manually test.

Reply:
@jmif Made this logic include teachers (and did a De Morgan transformation as I think it is easier to understand).
Does this look good to you?